src/emailmessagelistmodel.cpp

```diff
@@ -305,6 +306,13 @@ QVariant EmailMessageListModel::data(const QModelIndex & index, int role) const
     else if (role == MessageSenderFirstCharRole) {
         QString sender = messageMetaData.from().name();
         return firstChar(sender);
+    } else if (role == MessageParsedSubject) {
+        // Filter <img> and <ahref> html tags to make the text suitable to be displayed in a qml
+        // label using StyledText(allows only small subset of html)
+        QString subject = data(index, QMailMessageModelBase::MessageSubjectTextRole).toString();
+        subject.replace(QRegExp("<\\s*img", Qt::CaseInsensitive), "<no-img");
```

Comment:
Damn, that's annoying.  If anyone uses a subject line like "Project xyz is <abandoned>" it will get replaced...